content/blog/2025-01-05-xmpp-summit-27.md

```diff
@@ -0,0 +1,81 @@
+---
+title: XMPP Summit 27
+date: 2025-01-05
```

Comment:
This post should be released as soon as possible, and the text should be adapted accordingly (i.e. the summit being next year, 2025).

Reply:
7bd117d